src/lib/vm/vm.js

```diff
@@ -911,6 +911,7 @@ class VmStack {
           {
             src: this.vm.widgetSrc,
             type: StorageType.Private,
+            networkId: this.vm.widgetConfigs.findLast(config => config && config.networkId)?.networkId || this.vm.near.config.networkId
```

Comment:
Refactor this and below, by computing `networkId` once in the VM. Or by passing `networkId` to the VM from `Widget`

Reply:
Done ✅